GLOSSARY.md

```diff
@@ -2,6 +2,101 @@
 
 Alphabetically ordered list of terms defined through MGs.
 
-| Term          | Short Definition               | #MG                                                                 |
-|---------------|--------------------------------|-------------------------------------------------------------------------------------------------------|
-|         |     |       |
+**AB - Atomic Bridge**
+A bridge design that is atomic in the sense that a transfer either completes or not at all. > [MG-?]()
+
+***???? why atomic??. any transfer should be atomic***
```

Comment:
Do we actually consider it atomic? Because it's only atomic if with some assumptions. It could not be atomic if Relayer goes down and we are unable to bring it back in time.

Reply:
this is a valid point. @l-monninger wdyt on renaming the bridge to "native bridge"